pkg/mbedtls/contrib/entropy/entropy_sources_mbedtls_riot.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_mbedtls
```

Comment:
```suggestion
 * @ingroup     pkg_mbedtls_entropy
```
?

Reply:
Yes, probably a good idea to spend an own doxygen group now that we moved the entropy support to its own module.